tmail-backend/jmap/extensions/src/test/scala/com/linagora/tmail/james/jmap/contact/EmailAddressContactSearchEngineContract.scala

```diff
@@ -37,6 +37,11 @@ object EmailAddressContactSearchEngineContract {
   private val otherMailAddressUpperCase: MailAddress = new MailAddress("JOHNDOE@OTHER.COM")
   private val otherContactFieldsWithUppercaseEmail: ContactFields = ContactFields(otherMailAddressUpperCase, otherFirstname, otherSurname)
 
+  private val mailAddressRene: MailAddress = new MailAddress("rcordier@linagora.com")
```

Comment:
I don't think should advertise a real email address in the code though...

Reply:
sure, i will fix this